docs/topic-guides/session-basics.rst

```diff
@@ -84,10 +84,10 @@ configuration options.
 |                              | especially useful if your latest commit's build failed (see below) or you have unsaved    |
 |                              | work that was automatically recovered.                                                    |
 +------------------------------+-------------------------------------------------------------------------------------------+
-| Default Image                | This provides information about the Docker image used by the Session.                     |
+| Docker Image                 | This provides information about the Docker image used by the Session.                     |
```

Comment:
this is not specific to this change but this table is really hard to read (note the long scrollbar): 

![image](https://user-images.githubusercontent.com/3353586/167865938-f6cc9aa8-4a31-4c27-9614-0f254cfce3d1.png)

Reply:
True. I guess that can be only addressed in the CSS.
I'll add a CSS rule that changes the output just for this table. Let me know if you think we should extend it to all tables.

Reply:
isn't there some markdown option to word-wrap?

Reply:
I quickly googled the available solutions for `rst` files, and CSS seems the only way.

Reply:
looks good